.ostree/get_ostree_data.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+set -euo pipefail
+
+role_collection_dir="${ROLE_COLLECTION_DIR:-fedora/linux_system_roles}"
+ostree_dir="${OSTREE_DIR:-"$(dirname "$(realpath "$0")")"}"
+
+if [ -z "${4:-}" ]; then
+    cat <<EOF
+Usage: $0 packages [runtime|testing] DISTRO-MAJOR[.MINOR] [json|yaml|raw|toml]
+The script will use the packages and roles files in $ostree_dir to
+construct the list of packages needed to build the ostree image.  The script
+will output the list of packages in the given format
+- json is a JSON list like ["pkg1","pkg2",....,"pkgN"]
+- yaml is the YAML list format
+- raw is the list of packages, one per line
+- toml is a list of [[packages]] elements as in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/composing_installing_and_managing_rhel_for_edge_images/index#creating-an-image-builder-blueprint-for-a-rhel-for-edge-image-using-the-command-line-interface_composing-a-rhel-for-edge-image-using-image-builder-command-line
```

Comment:
With toml, the script returns packages as the following:

```
[spetrosi@fedora network]$ sh .ostree/get_ostree_data.sh packages runtime CentOS-8 toml
[[packages]]
name = "NetworkManager"
version = "*"
[[packages]]
name = "NetworkManager-team"
version = "*"
[[packages]]
name = "NetworkManager-wifi"
version = "*"
[[packages]]
name = "network-scripts"
version = "*"
[[packages]]
name = "nmstate"
version = "*"
[[packages]]
name = "python3-gobject-base"
version = "*"
[[packages]]
name = "wpa_supplicant"
version = "*"
```

Just asking, is it ok that each package is within a new [[packages]] section?

Reply:
yes - that is the TOML list-of-objects syntax - it looks weird but apparently that is the way TOML does it